src/commands/install.rs

```diff
@@ -43,29 +51,64 @@ impl InstallSubcommand {
         let mut package_sources = PackageSourceMap::new(default_registry);
         package_sources.add_fallbacks()?;
 
-        let mut try_to_use = BTreeSet::new();
-        for package in lockfile.packages {
-            match package {
-                LockPackage::Registry(registry_package) => {
-                    try_to_use.insert(PackageId::new(
-                        registry_package.name,
-                        registry_package.version,
-                    ));
-                }
-                LockPackage::Git(_) => {}
+        let try_to_use = lockfile.as_ids().collect();
+
+        let progress = ProgressBar::new(0).with_style(
+            ProgressStyle::with_template("{spinner:.cyan}{wide_msg}")?.tick_chars("⠁⠈⠐⠠⠄⠂ "),
+        );
+
+        progress.enable_steady_tick(Duration::from_millis(100));
+
+        if self.locked {
+            progress.println(format!(
+                "{} Verifying {}lockfile is up-to-date...",
+                SetForegroundColor(Color::DarkGreen),
+                SetForegroundColor(Color::Reset)
+            ));
+
+            let latest_graph = resolve(&manifest, &BTreeSet::new(), &package_sources)?;
+
+            if try_to_use != latest_graph.activated {
+                progress.finish_and_clear();
+
+                let old_dependencies = &try_to_use;
+
+                let changes = generate_depedency_changes(old_dependencies, &latest_graph.activated);
```

Comment:
```suggestion
                let changes = generate_dependency_changes(old_dependencies, &latest_graph.activated);
```
This was spelled wrong before too, I just think we can fix it while we're moving it around.